viofs/svc/virtiofs.c

```diff
@@ -390,6 +399,8 @@ static NTSTATUS VirtFsCreateFile(VIRTFS *VirtFs,
         create_in.create.flags |= O_APPEND;
     }
 
+    create_in.create.flags |= O_EXCL;
```

Comment:
But what if Windows had CREATE_ALWAYS specified to CreateFile? 

According to https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-createfilea:
 Creates a new file, always. If the specified file exists and is writable, the function overwrites the file, the function succeeds, and last-error code is set to ERROR_ALREADY_EXISTS (183). If the specified file does not exist and is a valid path, a new file is created, the function succeeds, and the last-error code is set to zero

Reply:
I'm not 100% sure but I think winfsp-tests cover this case and it is handled currently.